images/scripts/fedora.setup

```diff
@@ -50,6 +50,7 @@ udisks2-iscsi \
 
 COCKPIT_DEPS="\
 criu-libs \
+dnf5daemon-server \
```

Comment:
I *think* that even exists on Fedora 41, it's just not being used yet (as the default dnf is still 4)